pages/common/sqlmap.md

```diff
@@ -1,7 +1,7 @@
 # sqlmap
 
 > Detect and exploit SQL injection flaws.
-> More information: <https://sqlmap.org>.
+> More information: <http://sqlmap.org>.
```

Comment:
I've opened an issue here about this: https://github.com/sqlmapproject/sqlmap/issues/4638

Reply:
Fixed! https://github.com/sqlmapproject/sqlmap/issues/4638#issuecomment-824611258
```suggestion
> More information: <https://sqlmap.org>.
```